src/main/resources/templates/home.html

```diff
@@ -3,84 +3,118 @@
+          <h3>Fields in this register:</h3>
+          <div class="field-list">
+            <ul>
+              <li th:each="field : ${register.fields}" th:text="${field}"></li>
+            </ul>
+          </div>
 
-                <div><a th:href="${linkToRegisterRegister}">View this record in the register of registers</a></div>
-            </div>
-            <div class="column-third">
-                <ul>
-                    <li><a href="/download">Download data</a></li>
-                    <li th:if="${#bools.isTrue(contentPages.registerHistoryPageUrl.isPresent())}"><a th:href="${contentPages.registerHistoryPageUrl.get()}">Register history</a></li>
-                    <li><a th:href="${contentPages.registersIntroductionUrl}">About registers</a></li>
-                    <li><a th:href="${contentPages.usingRegistersGuidanceUrl}">Guidance for using registers</a></li>
-                    <li><a th:href="${contentPages.specificationUrl}">Technical specification</a></li>
-                </ul>
-            </div>
+          <a th:href="${linkToRegisterRegister}">View this record in the register of registers</a>
+        </div>
+        <div class="column-third">
+          <ul>
+            <li>
+              <a href="/download">Download data</a>
+            </li>
+            <li th:if="${#bools.isTrue(contentPages.registerHistoryPageUrl.isPresent())}">
+              <a th:href="${contentPages.registerHistoryPageUrl.get()}">Register history</a>
+            </li>
+            <li>
+              <a th:href="${contentPages.registersIntroductionUrl}">About registers</a>
+            </li>
+            <li>
+              <a th:href="${contentPages.usingRegistersGuidanceUrl}">Guidance for using registers</a>
+            </li>
+            <li>
+              <a th:href="${contentPages.specificationUrl}">Technical specification</a>
+            </li>
+          </ul>
         </div>
+      </div>
     </div>
 
     <details>
-        <summary>View register fingerprint</summary>
-        <div class="fingerprint-container">
-            <span class="fingerprint-explanation"><a href="https://gdstechnology.blog.gov.uk/2015/10/13/guaranteeing-the-integrity-of-a-register/">See how the fingerprint works</a></span>
-            <pre class="fingerprint"><code>{
-  "tree_size": 9803348,
-  "timestamp": 1447421303202,
-  "sha256_root_hash": "JATHxRF5gczvNPP1S1WuhD8jSx2bl+WoTt8bIE3YKvU=",
-  "tree_head_signature":
-  "BAMARzBFAiEAkKM3aRUBKhShdCyrGLdd8lYBV52FLrwqjHa5/YuzK7ECIFTlRmNuKLqbVQv0QS8nq0pAUwgbilKOR5piBAIC8LpS"
-}</code></pre>
-        </div>
+      <summary>View register fingerprint</summary>
+      <div class="fingerprint-container">
+        <span class="fingerprint-explanation">
+          <a href="https://gdstechnology.blog.gov.uk/2015/10/13/guaranteeing-the-integrity-of-a-register/">
+            See how the fingerprint works
+          </a>
+        </span>
+        <pre class="fingerprint">
+          <code>
```

Comment:
Unfortunately this change has caused an unexpected change in the "register fingerpint" section. This appears to be an annoying way that Tymeleaf works, where whitespace makes a difference to appearance. It might be worth changing this back to how it was - I know it is a bit of a hack, but seeing as we want to build a new front end at some point, I wouldn't want to waste too much time fixing this.

Now looks like this:

<img width="1001" alt="screen shot 2016-12-23 at 12 18 33" src="https://cloud.githubusercontent.com/assets/16761910/21453822/f571ba04-c90a-11e6-8d1d-9c241a45b924.png">


Should look like this:
<img width="1031" alt="screen shot 2016-12-23 at 12 18 50" src="https://cloud.githubusercontent.com/assets/16761910/21453809/e421b36c-c90a-11e6-9912-f8965e4c3c6a.png">